uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -285,11 +285,23 @@ function tasksAssignments_civicrm_alterAngular(\Civi\Angular\Manager $angular) {
     _tasksAssignments_change_workflow_help_text($doc);
     _tasksAssignments_remove_non_civihr_tabs_from_workflow($doc);
     _tasksAssignments_allow_only_add_timeline_action($doc);
+
+    $doc->find('.crmCaseType')
+      ->attr('ng-controller', 'CaseTypeExtendedController');
   });
 
   $angular->add($changeSet);
 }
 
+function tasksAssignments_civicrm_angularModules(&$angularModules) {
+  $angularModules['crm-tasks-workflows'] = array(
+    'ext' => 'civicrm',
+    'js' => array(
+      'tools/extensions/civihr/org.civicrm.reqangular/dist/reqangular.min.js',
+      'tools/extensions/civihr_tasks/uk.co.compucorp.civicrm.tasksassignments/js/dist/crm-tasks-workflows.min.js')
```

Comment:
I'm still not sure how to match your answer [here](https://github.com/compucorp/civihr-tasks-assignments/pull/335#discussion_r180702249) with the fact that we are not simply doing

```php
$angularModules['crm-tasks-workflows'] = array(
  'ext' => 'civicrm',
  'js' => array(
    '<path>/js/crm-tasks-workflows.js',
     '<path>/js/crm-tasks-workflows/*'
  )
);
```
*(in your answer you talk about defining a single module instead of multiples, but i was just talking about referencing the min file)*

why isn't the above correct?

Reply:
In core, depending upon the debug flag, they either use the globs or they minify all files. That works as they dont use any module loader like requirejs.

As we are using requirejs to bundle all of our code, we need to do the minification ourselves using `r.js`.